src/util.js

```diff
@@ -1,7 +1,10 @@
+'use strict'
+
 var multihashing = require('multihashing')
+var isNode = require('detect-node')
 
 exports = module.exports
 
 // Hash is the global IPFS hash function. uses multihash SHA2_256, 256 bits
-exports.hash = (data) => { return multihashing(data, 'sha2-256') }
-exports.isBrowser = () => { return !!global.window }
+exports.hash = (data) => multihashing(data, 'sha2-256')
+exports.isBrowser = () => !isNode
```

Comment:
one dependency to solve real world issues: https://github.com/ipfs/js-ipfs-api/issues/154